apps/settings/elements/developer_hud.html

```diff
@@ -69,6 +69,16 @@ <h2 data-l10n-id="hud-performance"></h2>
           </gaia-checkbox>
         </li>
         <li>
+          <p data-l10n-id="reflow-threshold"></p>
+          <span class="button icon icon-dialog">
+            <select name="devtools.reflowduration.threshold" data-value-type="integer">
+              <option value="1" selected>No Threshold</option>
```

Comment:
I think that some incremental reflows could happen in less than a millisecond. Please use `value="0"` as "No Threshold" (also, I'm not sure that the "No Threshold" string here can be localized, so please use "0" instead).